modules/installation-aws-config-yaml.adoc

```diff
@@ -312,7 +312,16 @@ endif::gov,secret,china[]
 ifdef::gov,secret,china[]
 <1> Required.
 endif::gov,secret,china[]
-<2> Optional: Add this parameter to force the Cloud Credential Operator (CCO) to use the specified mode, instead of having the CCO dynamically try to determine the capabilities of the credentials. For details about CCO modes, see the _Cloud Credential Operator_ entry in the _Red Hat Operators reference_ content.
+<2> Optional: Add this parameter to force the Cloud Credential Operator (CCO) to use the specified mode. By default, the CCO uses the root credentials in the `kube-system` namespace to dynamically try to determine the capabilities of the credentials. For details about CCO modes, see _About the Cloud Credential Operator_ in the _Authentication and authorization_ content.
++
+[IMPORTANT]
+====
+Setting this parameter to `Manual` enables alternatives to storing administrator-level secrets in the `kube-system` project, which require additional configuration steps:
+
+* Using the CCO utility (`ccoctl`) to implement short-term, limited-privilege security credentials for individual components. For more information, see _Configuring an AWS cluster to use short-term credentials_.
+
+* Managing cloud credentials manually. For more information, see _Manually creating long-term credentials_.
+====
```

Comment:
QE approved this as helpful, so I want to leave it alone for now. When I do Azure, I will look at moving it to the params table as a footnote (I think that's a great place for it), but that way it can get a clean ack in a different PR :sweat_smile:

Reply:
Here: https://github.com/openshift/openshift-docs/pull/64001/commits/4a47154ed3d69b4da779f67d3739d0dd792e91ca